core/src/main/java/io/kestra/core/models/flows/FlowWithException.java

```diff
@@ -39,4 +79,10 @@ public static Optional<FlowWithException> from(JsonNode jsonNode, Exception exce
         // if there is no id and namespace, we return null as we cannot create a meaningful FlowWithException
         return Optional.empty();
     }
+
+    /** {@inheritDoc} **/
+    @Override
+    public Flow toFlow() {
```

Comment:
I think we should remove the `toFlow()` method or at least deprecate it.
It seems bad practice when you mix FlowWithSource and Flow.